include/robot_state_publisher/joint_state_listener.h

```diff
@@ -69,7 +69,7 @@ class JointStateListener {
 
 
   /// Destructor
-  ~JointStateListener();
+  virtual ~JointStateListener();
```

Comment:
This makes sense, but I think making it virtual will break ABI. We try to avoid breaking ABI in released distros. Would you be willing to make the destructor protected instead?